apps/obs-web/src/views/DebrieferAssessment.vue

```diff
@@ -93,15 +98,51 @@ export default createComponent({
     const questions: any = ref([]);
     const answerSets: any = ref([]);
 
+    const samplingProcedures: any = ref([]);
+    const catchCategories: any = ref([]);
+
+    const jp = require('jsonpath');
+
+    const getTripProcedures: any = async () => {
+        let operationIDs = jp.query(debriefer.trips, '$..operationIDs');
+        operationIDs = flatten(operationIDs);
+        const operations = await masterDB.view(
```

Comment:
hey just fyi, I'm doing a similar query here to get all operations and I save the results in state. So you could just use state.debriefer.operations here rather than having to query couch. That codes is run when loading the trips data page which should always get run before a user navigates to the assessment page I believe. 

https://github.com/nwfsc-fram/boatnet/blob/master/apps/obs-web/src/views/DebrieferTrips.vue#L476